frontend/src/components/Notification.tsx

```
@@ -1,15 +1,15 @@
-        🎉 WR.DO Beta Launching Now!{" "}
-        <a
-          target="_blank"
-          className="text-blue-500 ml-2 underline after:content-['_↗']"
-          href="https://wr.do">
-          Try it
-        </a>
-      </p>
-    </div>
-  );
-}
+export default function Notifications({}: {}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)